### ISSUE TITLE
Typo in "Umbraco.ModelsBuider"  8.13.0 release of IsExternalModelsBuilderInstalled

### ISSUE THREAD

**Issue by mistyn8** [umbraco/Umbraco-CMS#10186]
Reverting to Umbraco.ModelsBuilder rather than embedded fails in 8.13.0 due to a typo

https://github.com/umbraco/Umbraco-CMS/blob/d428a4543f33bb7094cf7db5f6b6fdc2d1de3063/src/Umbraco.ModelsBuilder.Embedded/Compose/ModelsBuilderComposer.cs#L50

**Comment by mistyn8** [comment]
Knowing that I'm using Umbraco.ModelsBuilder.. quickly fixed this with a composer replacement.

```
using System;
using System.Reflection;
using Umbraco.Core;
using Umbraco.Core.Composing;
using Umbraco.Core.Logging;
using Umbraco.ModelsBuilder.Embedded.Configuration;
using Umbraco.Web;
using Umbraco.Web.PublishedCache.NuCache;

namespace Umbraco.ModelsBuilder.Embedded.Compose
{
    [ComposeBefore(typeof(NuCacheComposer))]
    [RuntimeLevel(MinLevel = RuntimeLevel.Run)]
    [Disable(typeof(ModelsBuilderComposer))]
    public sealed class UmbracoModelsBuilderComposer : IComposer
    {
        public void Compose(Composition composition)
        {
            composition.Configs.Add<IModelsBuilderConfig>(() => new ModelsBuilderConfig());

            if (IsExternalModelsBuilderInstalled() == true)
            {
                ComposeForExternalModelsBuilder(composition);
                return;
            }
        }

        private static bool IsExternalModelsBuilderInstalled()
        {
            var assemblyNames = new[]
            {
                "Umbraco.ModelsBuilder",
                "ModelsBuilder.Umbraco"
            };

            try
            {
                foreach (var name in assemblyNames)
                {
                    var assembly = Assembly.Load(name);

                    if (assembly != null)
                    {
                        return true;
                    }
                }
            }
            catch (Exception)
            {
                //swallow exception, DLL must not be there
            }

            return false;
        }

        private void ComposeForExternalModelsBuilder(Composition composition)
        {
            composition.Logger.Info<UmbracoModelsBuilderComposer>("ModelsBuilder.Embedded is disabled, the external ModelsBuilder was detected.");
            composition.Components().Append<DisabledModelsBuilderComponent>();
            composition.Dashboards().Remove<ModelsBuilderDashboard>();
        }
    }
}
```

**Comment by nul800sebastiaan** [comment]
Ah good catch 🤦‍♂️ - we'll get this fixed for 8.13.1! 👍

**Comment by nul800sebastiaan** [comment]
Fixed in https://github.com/umbraco/Umbraco-CMS/pull/10187

**Comment by flyinmryan** [comment]
How might someone go about implementing this embedded fix if they're using 8.12.4 and upgrading is not possible due to time constraints and other dev team's usage?  Thanks
